File: day2/lonely-integer/solution.py
a = [1,1,2,2,4,4,3]

def lonelyinteger(a):
    a.sort()
    for i in range(0,len(a),2):
        if i == len(a) - 1:
            return a[i]
        elif a[i] != a[i+1]:
            return a[i]
        
print(lonelyinteger(a))        

File: day2/lonely-integer/solution.js
let a = [0, 1, 0];

function lonelyinteger(a) {
  a.sort((a, b) => a - b);
  let size = a.length;
  for (let i = 0; i < size; i += 2) {
    console.log(i);
    if (a[i + 1] == undefined) {
      return a[i];
    } else if (a[i] != a[i + 1]) {
      return a[i];
    }
  }
}

console.log(lonelyinteger(a));
